### ISSUE TITLE
Cancellation notice sent with DOFFIN ID instead of TED ID

### ISSUE THREAD

**Issue by jlouvrier** [anskaffelser/eforms-sdk-nor#66]
A user referred to DOFFIN ID (4944-2024) instead of TED ID as OPP-090-Procedure Previous notice ID in a 29 CAN cancellation notice Notice. As a result, the contract notice does not show as canceled in DOFFIN.

CN: b79eba71-52f8-400a-bc51-8fb7a9453936
CAN: 70cf6f27-6b61-403f-9765-37a58100d3a4

Anything you can do on your end?

**Comment by kjorlaug** [comment]
I don't think so. The only suggestion is to either send a change correcting the 29 CAN, or if that fails send a new one with correct id (and then I'll contact TED).

**Comment by MaxKholoshnyi** [comment]
Just to doule check. Cusotmer need to create another form 29 but refference "2024-100100", right?
![image](https://github.com/anskaffelser/eforms-sdk-nor/assets/152489983/96be1b6f-884e-438f-9eb9-7dfdc12ae241)
